plenum/persistence/req_id_to_txn.py

```diff
@@ -11,30 +11,36 @@ class ReqIdrToTxn:
     """
 
     def __init__(self, keyValueStorage: KeyValueStorage):
+        self.delimiter = "~"
         self._keyValueStorage = keyValueStorage
 
-    @staticmethod
-    def getKey(identifier, reqId):
-        h = sha256()
-        h.update(identifier.encode())
-        h.update(str(reqId).encode())
-        return h.digest()
-
-    def add(self, identifier, reqId, seqNo):
-        key = self.getKey(identifier, reqId)
-        self._keyValueStorage.put(key, str(seqNo))
+    def add(self, digest, ledge_id, seq_no):
+        self._keyValueStorage.put(digest, self._create_value(ledge_id, seq_no))
 
     def addBatch(self, batch):
-        self._keyValueStorage.setBatch([(self.getKey(identifier, reqId), str(
-            seqNo)) for identifier, reqId, seqNo in batch])
-
-    def get(self, identifier, reqId) -> Optional[int]:
-        key = self.getKey(identifier, reqId)
+        self._keyValueStorage.setBatch([(digest, self._create_value(ledge_id,
+                                                                    seq_no))
+                                        for digest, ledge_id, seq_no in batch])
+
+    def get(self, digest):
+        """
+        Return leger_id, seq_no of transaction that was a result
+        of last request with this digest
+        :param digest: digest of request
+        :return: leger_id, seq_no
+        """
         try:
-            val = self._keyValueStorage.get(key)
-            return int(val)
+            val = self._keyValueStorage.get(digest)
+            return self._parse_value(val.decode())
         except (KeyError, ValueError):
-            return None
+            return None, None
+
+    def _parse_value(self, val: string):
+        parse_data = val.split(self.delimiter)
+        return str(parse_data[0]), str(parse_data[1])
```

Comment:
So, we expected seqNo as int previously and not it's str. Does it break anything?

Reply:
Changed it to return seqNo as int, but I don't sure that it's a good idea. In fact, we store in a seqNoDB string. And in the origin version of code we convert it to int after get() just in case.